src/_buttons.scss

```diff
@@ -225,3 +225,78 @@
     }
   }
 }
+
+
+.button.is-icon-only-button {
+  background-color: transparent;
+  color: lighten($button-icon-only-default-color, 10.5%);
+  padding: 0.5em 0.25em;
+
+  line-height: 1;
+
+  font-size: 1em;
+  width: 2em;
+  height: 2em;
+  text-align: center;
+
+  &.is-small {
+    font-size: 0.9em;
+  }
+  &.is-medium {
+    font-size: 1.1em;
+  }
+  &.is-large {
+    font-size: 1.2em;
+  }
+  &.is-very-large {
+    font-size: 1.4em;
+  }
+  &.is-extremely-large {
+    font-size: 1.6em;
+  }
```

Comment:
I'd recommend adopting Bootstrap's approach to sizes here: XS, S, M, L, XL. It's more familiar, because it's the approach used on most things in the real world too. It also lends some conciseness: `.is-extremely-large` is long and a bit of a pain to type out, but `.is-xl` (or `.is-size-xl`, if you want to make it obvious that it's sizing) is nice and easy.

Reply:
I'll ignore this for now and consider it after merging this PR.